src/runners/baseRunner/BaseRunner.ts

```diff
@@ -180,6 +182,25 @@ export abstract class BaseRunner implements ITestRunner {
         return commandParams;
     }
 
+    protected getJavaEncoding(): string {
```

Comment:
These two methods are just changed modifier from `private` to `protected` and changed the location up